From f9a018e8a6af2898dc782f6e526bd11f6f352e87 Mon Sep 17 00:00:00 2001
Message-Id: <f9a018e8a6af2898dc782f6e526bd11f6f352e87.1638867301.git.geliang.tang@suse.com>
From: Yang Li <yang.lee@linux.alibaba.com>
Date: Sun, 11 Apr 2021 21:43:12 +0800
Subject: [PATCH] bcache: use NULL instead of using plain integer as pointer
Git-commit: f9a018e8a6af2898dc782f6e526bd11f6f352e87
Patch-mainline: v5.13-rc1
References: bsn#32
Modified-by-SEL: No


This fixes the following sparse warnings:
drivers/md/bcache/features.c:22:16: warning: Using plain integer as NULL
pointer

Reported-by: Abaci Robot <abaci@linux.alibaba.com>
Signed-off-by: Yang Li <yang.lee@linux.alibaba.com>
Signed-off-by: Coly Li <colyli@suse.de>
Link: https://lore.kernel.org/r/20210411134316.80274-4-colyli@suse.de
Signed-off-by: Jens Axboe <axboe@kernel.dk>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 drivers/md/bcache/features.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/md/bcache/features.c b/drivers/md/bcache/features.c
index d636b7b2d070..6d2b7b84a7b7 100644
--- a/drivers/md/bcache/features.c
+++ b/drivers/md/bcache/features.c
@@ -19,7 +19,7 @@ struct feature {
 static struct feature feature_list[] = {
 	{BCH_FEATURE_INCOMPAT, BCH_FEATURE_INCOMPAT_LOG_LARGE_BUCKET_SIZE,
 		"large_bucket"},
-	{0, 0, 0 },
+	{0, 0, NULL },
 };
 
 #define compose_feature_string(type)				\
-- 
2.31.1

